static/api/v2.yaml

```diff
@@ -1581,30 +1581,710 @@ paths:
+                    short_code_delivery_details:
+                      short_code: '123456789'
+                      expire_time: '2022-08-12T05:53:07.119Z'
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                Invalid Parameters:
+                  $ref: '#/paths/~1v2~1users/get/responses/400/content/application~1json/examples/Invalid%20Parameters'
+        '401':
+          description: Unauthorized.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                Missing Authorization:
+                  $ref: '#/paths/~1v2~1users/get/responses/401/content/application~1json/examples/Missing%20Authorization'
+        '403':
+          description: Forbidden.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                Insufficient Authorization:
+                  $ref: '#/paths/~1v2~1users/get/responses/403/content/application~1json/examples/Insufficient%20Authorization'
+        '404':
+          description: Not found.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                User Not Found:
+                  value:
+                    code: not_found
+                    message: user not found
+                    details:
+                      - type: ResourceInfo
+                        resource_type: User
+                        id: 37660157-aa4b-4e84-958f-1fdf27777e86
+                        description: user not found
+        '412':
+          description: Precondition Failed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                User Not Active:
+                  value:
+                    code: failed_precondition
+                    message: user not active
+                    details:
+                      - type: ResourceInfo
+                        resource_type: User
+                        id: 37660157-aa4b-4e84-958f-1fdf27777e86
+                        description: user not active
+        '500':
+          description: Server error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                Internal Error:
+                  $ref: '#/paths/~1v2~1users/get/responses/500/content/application~1json/examples/Internal%20Error'
+  /v2/passkeys:
+    get:
+      tags:
+        - Passkeys
+      operationId: GetPasskeys
+      summary: List Passkeys.
+      description: |
+        write a description about passkeys
```

Comment:
This needs the following parts of the description:
1) An actual description, returns all passkeys for a tenant. The tenantID is taken from the token.
2) We need a section for better documentation on the different operators that we support.
3) We are missing documentation for each of the parameters.